Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -1,594 +1,354 @@
+  background-color: var(--boclist-tablehead-background-color);
+  white-space: nowrap;
 }
 
-th.bocListTitleCell > a[href]:focus,
-th.bocListTitleCell > a[href]:hover
+div.bocList table.bocListTable th.bocListTitleCell > span,
+div.bocList table.bocListTable th.bocListTitleCell > a
 {
-  color: Black;
-  padding: 1px;
-  border: 1px solid #7CB2E6;
-  background-color: #E5F0FC;
-  outline: none;
+  display: inline-flex;
+  align-items: center;
+  height: var(--remotion-themed-min-height);
+  margin: 3px 0;
 }
 
-th.bocListTitleCell > a[href]:active
+div.bocList table.bocListTable th.bocListTitleCell > a,
+div.bocList table.bocListTable tr.bocListDataRow td.bocListDataCell > a
 {
-  color: Black;
-  border: solid 1px #569CE0;
-  background-color: #C5E0FC;
+  color: var(--color-primary);
 }
 
-th.bocListTitleCell .bocListSortingOrder img
+div.bocList table.bocListTable th.bocListTitleCell > a[href]:active,
+div.bocList table.bocListTable tr.bocListDataRow td.bocListDataCell a:active
 {
-  height: 16px;
-  width: 16px;
+  color: black;
 }
 
-th.bocListTitleCell > a[href]:focus .bocListSortingOrder img,
-th.bocListTitleCell > a[href]:hover .bocListSortingOrder img,
-th.bocListTitleCell > a[href]:active .bocListSortingOrder img
+div.bocList table.bocListTable th.bocListTitleCell > a[href]:focus > :first-child,
+div.bocList table.bocListTable th.bocListTitleCell > a[href]:hover > :first-child,
+div.bocList table.bocListTable tr.bocListDataRow td.bocListDataCell > a:focus,
+div.bocList table.bocListTable tr.bocListDataRow td.bocListDataCell > a:hover
 {
-  -webkit-filter: brightness(0);
-  filter: brightness(0);
-  filter: progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
+  text-decoration: underline;
 }
 
-/*  Style applied to the cells in the BocList's odd and even data rows. */
-td.bocListDataCell
+div.bocList table.bocListTable tr.bocListDataRow td.bocListDataCell
 {
-  text-align: left;
-
-  padding: 2px 5px 2px 2px;
-  border-top: none;
-  border-right: none;
-  border-left: none;
-  border-bottom: 1px solid #C8C8C8;
-
-  cursor: default;
+  background-color: white;
 }
 
-/*  Style applied to the cells in the BocList's not-selected odd and even data rows. */
-tr.bocListDataRow.odd td.bocListDataCell
+div.bocList table.bocListTable tr.bocListDataRow.selected td.bocListDataCell
 {
-  background-color: White;
+  background-color: var(--color-highlight);
 }
 
-tr.bocListDataRow.even td.bocListDataCell
+div.bocList table.bocListTable td.bocListDataCell
 {
-  background-color: #F0F0F0;
+  border-bottom: 1px solid var(--boclist-tablecell-border-color);
 }
 
-/*  Style applied to the cells in the BocList's selected odd and even data rows. */
-tr.bocListDataRow.selected td.bocListDataCell
-{
-  background-color: #C5E0FC;
-}
-
-
-/*  Style applied to the cell in the BocList's data row that contains the index. */
-th.bocListTitleCellIndex
+div.bocList table.bocListTable th.bocListTitleCellIndex,
```

Comment:
Why is the `bocListDataCellIndex` centered anyway?

Reply:
Hmm... yeah, we should do a right-aligned text here instead of centered. Left-aligned is wrong because it's a counter and that may get more digits as it counts on. Centered was probably chosed because it looked better when the title text was longer than the counter-digits. but I guess we can just use right-aligned.